scripts/update/update_disease_ontology_terms.py

```diff
@@ -0,0 +1,288 @@
+                    print("Ontologies updated successfully:", response_json)
+                    if "errors" in response_json:
+                        for error in response_json["errors"]:
+                            print(f"ERROR: {error}")
+                else:
+                    print("Failed to update ontologies:", response_update.status_code, response_update.json())
+            except Exception as e:
+                print("Error:", e)
+        else:
+            print("Error: cannot login into G2P")
+    else:
+        print("INFO: no ontology terms to update")
+
+
+def delete_ontologies(records_to_delete: list[str], api_username: str, api_password: str, api_url: str) -> None:
+    """
+    Method that calls the G2P API to delete a list of ontology accession.
+
+    Args:
+        records_to_delete (list[str]): list of ontology accessions to delete
+        api_username (str): G2P API username
+        api_password (str): G2P API password
+        api_url (str): G2P API URL
+    """
+
+    ontology_url = "update/disease_ontology_terms/"
+    login_url = "login/"
+
+    data = {
+        "username": api_username,
+        "password": api_password
+    }
+
+    if records_to_delete:
+        # Login
+        response = requests.post(api_url + login_url, json=data)
+        if response.status_code == 200:
+            try:
+                # Delete ontology terms
+                response_delete = requests.delete(api_url + ontology_url, json=records_to_delete, cookies=response.cookies)
+                if response_delete.status_code == 200:
+                    response_json = response_delete.json()
+                    print("Ontologies delete successfully:", response_json)
+                    if "errors" in response_json:
+                        for error in response_json["errors"]:
+                            print(f"ERROR: {error}")
+                else:
+                    print("Failed to delete ontologies:", response_delete.status_code, response_delete.json())
+            except Exception as e:
+                print("Error:", e)
+        else:
+            print("Error: cannot login into G2P")
+    else:
+        print("INFO: no ontology terms to delete")
+
+
+def main():
+    """
+    Script to update the Mondo/OMIM ontology terms.
+    """
+
+    parser = argparse.ArgumentParser(description="")
+    parser.add_argument("--config", required=True, help="Config file")
+    parser.add_argument("--api_username", required=True, help="Username to connect to the G2P API")
+    parser.add_argument("--api_password", required=True, help="Password to connect to the G2P API")
+    parser.add_argument("--dryrun", required=False, default=0, help="Option to test update")
+    args = parser.parse_args()
+
+    config_file = args.config
+    dryrun = args.dryrun
+    api_username = args.api_username
+    api_password = args.api_password
+
+    # Load the config file
+    config = configparser.ConfigParser()
```

Comment:
Suggesting that this is a different function

Reply:
Like from L258 TO 2666

Reply:
A separate function or should we use another function to fetch the config data?

Reply:
No a separate function, just the modularity thing.

Reply:
Can you give an example? I don't understand the comment.

Reply:
ah ok, a separate function.
We read the config file in the main function in all our scripts - that is why I added this update here.
If we update this code to read the db details from a separate function we should first create a jira ticket and do the update in all scripts at the same time. That way the new implementation is done in the whole repo.

Reply:
Yeah but if we already have it in this, less to add. To even keep it dry, we can have a file in our scripts repo, just to read configuration and it can have this detail and we just have to import and stuff

Reply:
Yeah but it is reusable if you keep it in a separate file. So we just have to call it once in the different scripts. 
Also again for testing for the future. Also allows for easier debug